### DISCUSSION TITLE
Tailwind v3.0.11 not reflecting on my react project.

### DISCUSSION THREAD

**Discussion by sancodes** [tailwindlabs/tailwindcss#6942] [Help] [answered]
My initial installation as per the [documentation](https://tailwindcss.com/docs/guides/create-react-app) was successful. 
However, 

Even after multiple research and modification, the Tailwind does not reflect on my react project. 

Closest Stackoverflow issue I was able to go through was [this](https://stackoverflow.com/questions/70313432/tailwindcss-v3-0-1-classes-is-not-working)

my package.json currently has that dependencies in the app: 
```
{
  "name": "portfolio",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@testing-library/jest-dom": "^5.16.1",
    "@testing-library/react": "^12.1.2",
    "@testing-library/user-event": "^13.5.0",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-scripts": "5.0.0",
    "web-vitals": "^2.1.2"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "autoprefixer": "^10.4.1",
    "postcss": "^8.4.5",
    "tailwindcss": "^3.0.11"
  }
}

```

I have tried switching the **react-scripts** version to _"5.0.0-next.58"_ 
and **tailwind** version to _"^0.0.0-insiders.e6e24b8"_ 
as per the guy who mentioned in stackoverflow, still no luck. 

My node version is **v17.3.0**

**Comment by NovusOpus** [comment, marked as answer]
Maybe you didn't setup the [`content`](https://tailwindcss.com/docs/content-configuration) correctly?

You need to include the paths of the folders which contains your template files(html,js,jsx,ts,tsx) that you're adding TailwindCSS Classes or Utilities.

So for example if you using tailwind in files inside `myapp` folder you should include that folder path as well like this:
```js
module.exports = {
  content: [
    "./src/**/*.{js,jsx,ts,tsx}",
    "./myapp/**/*.{html,js,jsx,ts,tsx}",
  ],
  // ...
}
```

If that's not the case then can you share a repo to look at?

**Reply by sancodes** [reply]
The redirection towards the [content](https://tailwindcss.com/docs/content-configuration) was really helpful. Learned tons of useful way to include paths. Thank you for your time. :)
